Userland/Libraries/LibGUI/ValueSlider.cpp

```diff
@@ -119,9 +119,14 @@ Gfx::IntRect ValueSlider::bar_rect() const
     return bar_rect;
 }
 
+int ValueSlider::knob_size() const
```

Comment:
It's a bit confusing calling this "size" when "size" is elsewhere used to mean a width and height.

Reply:
length perhaps?

Reply:
Sure, that works. :+1: